Sources/Sentry/SentryBreadcrumbTracker.m

```diff
@@ -141,13 +141,31 @@ - (void)addEnabledCrumb
     [SentrySDK addBreadcrumb:crumb];
 }
 
+#if SENTRY_HAS_UIKIT
++ (BOOL)avoidSender:(id)sender forTarget:(id)target action:(NSString *)action
+{
+    if ([sender isKindOfClass:UITextField.self]) {
+        // This is required to avoid creating breadcrumbs for every key pressed in a text field.
+        // Textfield may invoke many types of event, in order to check if is a
+        // `UIControlEventEditingChanged` we need to compare the current action to all events
+        // attached to the control. This may cause a false negative if the developer is using the
+        // same action for different events.
```

Comment:
```suggestion
        // This is required to avoid creating breadcrumbs for every key pressed in a text field.
        // Textfield may invoke many types of event, in order to check if is a
        // `UIControlEventEditingChanged` we need to compare the current action to all events
        // attached to the control. This may cause a false negative if the developer is using the
        // same action for different events, but this trade off is acceptable because using the same
        // action for `.editingChanged` and another event is not supposed to happen.
```